daktari/options.py

```diff
@@ -14,6 +14,9 @@ def validate_as_file_path(parser: ArgumentParser, arg: str) -> Path:
 
 argument_parser = ArgumentParser(description="Check developer environment configuration.")
 argument_parser.add_argument("--debug", default=False, action="store_true", help="turn on debug logging")
+argument_parser.add_argument(
+    "--generate", default=False, action="store_true", help="generate a template file for local configuration"
```

Comment:
`generate` is maybe a bit vague, unless we're planning on adding more?

I don't think we need the default=False either if we're using "store_true", but maybe it makes it a bit more readable? https://docs.python.org/3/library/argparse.html#action

Reply:
Yeah, I was just trying to save on typing really. `--generate-local-config` felt clunky to ask people to type out. Open to suggestions, will change to the more verbose one if you prefer it as I'm like 50/50 :sweat_smile:

Reply:
oooh, how about `--generate-local-config` but also alias as `-g`? That's how tools usually do this sort of thing, I'll do that :smile:

Reply:
I think that sounds good 😄